core/kazoo/Makefile

```diff
@@ -41,6 +41,9 @@ $(GEN_ERL): $(GEN_FILE) $(GEN_ERL_SRC)
 		| sort -u -t' ' -k1,1 \
 		>> $(GEN_ERL)
 	@echo 'from_extension(<<"pem">>) -> <<"application/x-pem-file">>;' >> $(GEN_ERL)
+	@echo 'from_extension(<<"PDF">>) -> <<"application/pdf">>;' >> $(GEN_ERL)
+	@echo 'from_extension(<<"TIF">>) -> <<"image/tiff">>;' >> $(GEN_ERL)
+	@echo 'from_extension(<<"TIFF">>) -> <<"image/tiff">>;' >> $(GEN_ERL)
```

Comment:
maybe changing line 35 to `-> from_extension(kz_term:to_lower_binary(Ext))`  instead ?

Reply:
that works for me, a lot more flexible. I pushed that change.